modules/nw-operator-cr.adoc

```diff
@@ -296,6 +296,10 @@ This field cannot be changed after installation.
 |integer
 |The maximum size for the audit log in bytes. The default value is `50000000` or 50 MB.
 
+|`maxLogFiles`
+|integer
+|The maximum number of log files the audit log.
```

Comment:
@jab-rh Could we further clarify that these are log files that are retained or at a given time?
Something like this 
The maximum number of  ACL audit log files retained.

Reply:
@asood-rh, I've updated this.